include/deal.II/distributed/tria.h

```diff
@@ -809,6 +809,10 @@ namespace parallel
        * The vector can be filled by the function
        * GridTools::collect_periodic_faces.
        *
+       * For more information on periodic boundary conditions see
+       * GridTools::collect_periodic_faces, DoFTools::make_periodicity_constraints
+       * and @ref step_45 "step-45".
```

Comment:
@ref step_45 "step-45"       ->   step-45
(the substitution is done by the filter script)